### PR TITLE
BST-12323: reenable validating secrets by default in gitleaks

### PR DESCRIPTION
**Pull request by ledo01**
This update the keyscope post-processor version to enable validation the secrets by default.

### CODE REVIEW

**Comment by ledo01** [comment]
Tested locally, validate the secrets unless `GITLEAKS_VALIDATE_SECRETS=false` is specified.

**Review by SUSTAPLE117 — APPROVED** [review]

**Comment by ledo01** [comment]
I'll merge after: https://github.com/boostsecurityio/helm-releases/pull/12089 gets deploy to avoid blocking the release for other scanner.

**Pull request merged**
